controllers/storageclass/storageclass_controller.go

```diff
@@ -116,6 +119,11 @@ func (r *StorageClassWatcher) Reconcile(_ context.Context, request reconcile.Req
 		return result, err
 	}
 
+	flashSystemCluster, err := r.getFlashSystemClusterByStorageClass(sc)
+	if err != nil {
```

Comment:
if err != nil then we have an error and we need to return. maybe you mean err == nil?

Reply:
fixed